testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -228,6 +229,8 @@ public static String loadUserForTest(LabUserQuery query) {
     public static String loadTempUser(final String userType) {
         instance.setupApiClientWithAccessToken();
         CreateTempUserApi createTempUserApi = new CreateTempUserApi();
+        createTempUserApi.getApiClient().setConnectTimeout((int) TimeUnit.SECONDS.toMillis(15));
+        createTempUserApi.getApiClient().setReadTimeout((int) TimeUnit.SECONDS.toMillis(15));
```

Comment:
Can we make the value a CONSTANT?  Also, do we want a 15 *second* connect timeout?  That's a little absurd for things running in the same network.

Reply:
yes I can make it a constant.....and yes let me remove it for connect and just do it for read

Reply:
Addressed